cmd/docker-app/status.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"github.com/deislabs/duffle/pkg/action"
+	"github.com/deislabs/duffle/pkg/claim"
+	"github.com/deislabs/duffle/pkg/credentials"
+	"github.com/deislabs/duffle/pkg/utils/crud"
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/spf13/cobra"
+)
+
+func statusCmd(dockerCli command.Cli) *cobra.Command {
+	var opts credentialOptions
+
+	cmd := &cobra.Command{
+		Use:   "status <installation-name>",
+		Short: "Get the installation status. If the installation is a docker application, the status shows the stack services.",
+		Args:  cli.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runStatus(dockerCli, args[0], opts)
+		},
+	}
+	opts.addFlags(cmd.Flags())
+
+	return cmd
+}
+
+func runStatus(dockerCli command.Cli, claimName string, opts credentialOptions) error {
+	muteDockerCli(dockerCli)
+	h := duffleHome()
+
+	claimStore := claim.NewClaimStore(crud.NewFileSystemStore(h.Claims(), "json"))
+	c, err := claimStore.Read(claimName)
+	if err != nil {
+		return err
+	}
+	targetContext := getTargetContext(opts.targetContext, dockerCli.CurrentContext())
```

Comment:
We should check if the status custom action is present (it is no more part of the spec)

Reply:
Let's do that in a follow-up

Reply:
Fixed in #451